templates/emails/email-header.php

```diff
@@ -16,15 +16,15 @@
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly
+	exit; // Exit if accessed directly.
 }
 
 ?>
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=<?php bloginfo( 'charset' ); ?>" />
-		<title><?php echo get_bloginfo( 'name', 'display' ); ?></title>
+		<title><?php echo esc_attr( get_bloginfo( 'name', 'display' ) ); ?></title>
```

Comment:
This is the wrong use of escaping here. Should be `esc_html()`. I see other areas in this PR doing the same as well.